src/main/java/com/CezaryZal/manager/modifier/entity/ContainersCreatorOfUserHcAuth.java

```diff
@@ -0,0 +1,22 @@
+package com.CezaryZal.manager.modifier.entity;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ContainersCreatorOfUserHcAuth {
+
+    public List<String> getRoleList(String roles){
+        if(roles.length() > 0){
+            return Arrays.asList(roles.split(","));
+        }
+        return new ArrayList<>();
+    }
+
+    public List<String> getPermissionList(String permissions){
+        if(permissions.length() > 0){
+            return Arrays.asList(permissions.split(","));
+        }
+        return new ArrayList<>();
```

Comment:
Jak role będą w Set<?> a nie String to to będzie zbędne.

Reply:
Ogółem tak, jednak czy z db czy od użytkownika role będą zawsze w Stringu